### PR TITLE
Show FirstFt signup email component on Companies section page

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
Also rename references to `light-signup` (that's the general term for this component)

### CODE REVIEW

**Comment by andygnewman** [comment]
Always been a fan of skinny controllers, so wondering whether you can start using the 'transform' model that we have to do this, otherwise we're configuring different streams in different places and could get messy quite quickly.

Plus it makes test coverage, debugging a bit easier.

**Comment by i-like-robots** [comment]
Details about authoring a transform are in the wiki https://github.com/Financial-Times/next-stream-page/wiki/transforms

**Comment by ironsidevsquincy** [comment]
Will update

**Comment by ironsidevsquincy** [comment]
Updated (love this transform stuff btw)

**Comment by andygnewman** [comment]
Looking good. Just a couple of comments.

Putting the uuids in the transform rather than the profiles. I think this makes sense given that it's a very light transform and would add a lot more complexity to the profile config.

Using the transforms makes writing tests super easy (nudge, nudge).

**Comment by ironsidevsquincy** [comment]
Lol. I thought you meant easy for somebody _else_. 

Ok, ok, I was being lazy, tests on their way

**Comment by ironsidevsquincy** [comment]
Now with tests

File: test/server/transforms/light-signup.test.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+			const res = {
+				templateData: { },
+				locals: { anon: { userIsAnonymous: false } }
+			};
+			const lightSignupTransform = new (lightSignup());
+			lightSignupTransform.showLightSignup = true;
+			lightSignupTransform.post(res, { firstFtEmailSignup: true });
+			res.templateData.showLightSignup.should.be.false;
+		});
+
```
Comment (andygnewman) [inline test/server/transforms/light-signup.test.js]: Maybe the one missing scenario for completeness;
```
it('should not show if not enabled for this concept', () => {
    const res = {
        templateData: {},
        locals: { anon: { userIsAnonymous: true } }
    };
    const lightSignupTransform = new (lightSignup());
    lightSignupTransform.showLightSignup = false;
    lightSignupTransform.post(res, { firstFtEmailSignup: true });
    res.templateData.showLightSignup.should.be.false;
});
```

Comment (ironsidevsquincy) [inline test/server/transforms/light-signup.test.js]: Good point

**Comment by andygnewman** [comment]
👍  (sorry for making that more difficult, but worth it I think)

**Comment by ironsidevsquincy** [comment]
No problem, might as well do it properly

**Pull request merged**